game/card.go

```diff
@@ -34,15 +32,21 @@ type Card struct {
 	BasePower     int
 	BaseToughness int
 	BaseTrample   bool
+
+	// Properties that are relevant for Lands and other mana producers
+	Colorless         int
+	SacrificesForMana bool
 }
 
 //go:generate stringer -type=CardName
 type CardName int
 
 const (
-	Forest CardName = iota
+	EldraziSpawnToken CardName = iota
```

Comment:
This surprised me at first to consider a token a card, but i suppose it is reasonable